### PR TITLE
Final commit

### PR DESCRIPTION
**Pull request by nehalskdev**

### CODE REVIEW

**Comment by SaadMahi** [comment]
try to fix the height of home page in 100vh

**Review by anask601 — COMMENTED** [review]
@nehalskdev Unintended horizontal scrolling and empty white space from the right are happening.

**Comment by nehalskdev** [comment]
@anask601 please check is the white space gone?

File: index.html [hunk 1]
```diff
@@ -44,20 +44,23 @@
       <main>
         <h1>
           Your cat will go
-          <img src="bonkers.svg" alt="Text that says bonkers" />
+          <img src="assets/bonkers.svg" alt="Text that says bonkers" />
```
Comment (ManSOorcode) [inline index.html:47]: manage it's width according to small screen

**Review by ManSOorcode — CHANGES_REQUESTED** [review]

**Comment by SaadMahi** [comment]
LGTM 👍

**Pull request merged**